azext_edge/edge/params.py

```diff
@@ -1184,3 +1184,47 @@ def load_iotops_arguments(self, _):
             help="Fully qualified role definition Id in the following format: "
             "/subscriptions/{subscriptionId}/providers/Microsoft.Authorization/roleDefinitions/{roleId}",
         )
+
+    with self.argument_context("iot ops connector opcua trust") as context:
+        context.argument(
+            "instance_name",
+            options_list=["--instance", "-i"],
+            help="IoT Operations instance name.",
+        )
+        context.argument(
+            "instance_resource_group",
+            options_list=["--instance-resource-group", "--ig"],
+            help="Instance resource group.",
```

Comment:
is this different to the default `-g` param that we use? Like, is there both a `resource_group` and an `instance_resource_group`?